walkthroughs/tls-with-acm/infrastructure/ecs-service.yaml

```diff
@@ -72,7 +72,7 @@ Resources:
           - Name: 'ProxyEgressPort'
             Value: '15001'
           - Name: 'AppPorts'
-            Value: '9080'
+            Value: '80'
```

Comment:
Just for my own curiosity since I tend to lose track of which ports matter to which piece of infra: why change this?

Reply:
Setting custom ports isn’t necessary and just increases the cognitive load. I’d like our examples to start being as normal as possible (e.g. just use 80 for http).